docs/README.md

```diff
@@ -0,0 +1,89 @@
+# IPC Agent
+
+The IPC Agent is a process faciliting the participation of Filecoin clients like Lotus in the InterPlanetary Consensus (formerly Hierarchical Consensus).
+
+Please refer to the [IPD Agent Design](https://docs.google.com/document/d/14lkRRv6MQYnuEfp2GoGngdD8Q5YgfE38D8HTZWKgKf4) document for details on the agent.
+
+
+# IPLD Resolver
+
+The [IPLD Resolver](../ipld/resolver) is a library that IPC Agents can use to exchange data between subnets in IPLD format.
+
+## Checkpointing
+
+The most typical use case would be the propagation of checkpoints from child subnets to the parent subnet.
```

Comment:
For M2 this is not quite the case, the resolver will be used to fetch cross-net messages.

Reply:
I may be misunderstanding what that means. I thought a cross-net message is a catch-all phrase for bottom-up and top-down messages. I understand that strictly they probably mean this:

```
/root |- /sub1  >
      |          |
      |          |
      |     cross-msg      
      |          |
      |          |
      |- /sub2  < 
```
But I thought that would be a combination of a bottom-up and a top-down message, and involve a checkpoint.